### ISSUE TITLE
Docs: Create 'Create' app help guide with hotkeys and functionality details then link to that from the 'Create' app

### ISSUE THREAD

**Issue by kasenvr** [vircadia/vircadia#880]

**Comment by AleziaKurdis** [comment]
For the Hotkeys
Here's the current exhaustive list:


ACTION | HOTKEY | INFO IN APP. MENU
-- | -- | --
Select All (in List) | Ctrl-A | Yes
Inverse Selection (in List) | Ctrl-I | Yes
Undo | Ctrl-Z | Yes
Redo | Ctrl-Y | Yes
Cut | Ctrl-X | Yes
Copy | Ctrl-C | Yes
Paste | Ctrl-V | Yes
Duplicate | Ctrl-D | Yes
Delete | Del | Yes
Parent Entities to the Last Selected | Ctrl-P | Yes
Unparent Entity | Ctrl-Shift-P | Yes
Set Camera Focus to Selection | Ctrl-F | Yes
Align Grid to Selected Entities | Ctrl-G | Yes
Toggle Local/World | Ctrl-T | **No**

**Comment by JulianGro** [comment]
With docs, I assume there docs are ment? https://github.com/kasenvr/vircadia-docs-sphinx/

For reference:
https://docs.vircadia.dev/explore/get-started/desktop.html?highlight=ctrl#create-and-edit-mode

**Comment by kasenvr** [comment]
We'd want to link to that from the 'Create' app then I imagine + make sure she's up to date.

**Comment by AleziaKurdis** [comment]
If we implement those 3 menus in the create app, 
then a link to the doc might not be necessary:
![image](https://user-images.githubusercontent.com/60075796/100304032-e4b4ea80-2f6b-11eb-9a0d-b76027a65556.png)

Unless the doc get enhanced with more than just the hotkeys list.

**Comment by JulianGro** [comment]
Yeah if I was to write a list in the docs, I would definitely write some more explaination to what stuff does and how to use it.
But your menu suggestion definitely gets rid of the need to add only the shortcuts to the docs. 
Anyways a doc update would likely take some time, as is am essentially eternally waiting for reviews ;P

**Comment by AleziaKurdis** [comment]
Documentation about the Create App. Hotkeys
will need to be updated after this PR:
 **#894 Create App. - Feature Bundle - Dec 2020**

**Comment by JulianGro** [comment]
:angry:
